src/index.html

```diff
@@ -4,10 +4,265 @@
+        </section>
+      </article>
+      <article class="main__category-box" id="categories">
+        <p class="main__category">Browse Bose <br>products by category</p>
+      </article>
+
+      <article class="main__products products" id="buy">
+        <section
+          class="products__card"
+          data-aos="fade-up"
+          data-aos-duration="1200"
+        >
+          <section class="products__card-pictures">
+            <div class="products__big-img products__big-img--1"></div>
+            <div class="products__small-img products__small-img--1"></div>
+          </section>
+          <p class="products__title">Headphones & earbuds</p>
+        </section>
+
+        <section
+          class="products__card"
+          data-aos="fade-up"
+          data-aos-duration="1200"
+        >
+          <section class="products__card-pictures">
+            <div class="products__big-img products__big-img--2"></div>
+            <div class="products__small-img products__small-img--2"></div>
+          </section>
+          <p class="products__title">Speakers</p>
+        </section>
+
+        <section
+          class="products__card"
+          data-aos="fade-up"
+          data-aos-duration="1200"
+        >
+          <section class="products__card-pictures">
+            <div class="products__small-img products__small-img--3"></div>
+            <div class="products__big-img products__big-img--3"></div>
+          </section>
+          <p class="products__title">Audio sunglasses</p>
+        </section>
+
+        <a href="#" class="products__button">View all</a>
+      </article>
+
+      <article
+        class="main__reasons reasons"
+        data-aos="fade-up"
+        data-aos-duration="1200"
+      >
+        <section class="reasons__box">
+          <section class="reasons__title-box">
+            <p class="reasons__title">Why buy direct <br>from Bose</p>
+          </section>
+          <section class="reasons__subtitle">
+            <p class="reasons__advantages">Free 2-day shipping and returns</p>
+            <p class="reasons__advantages">90-day risk-free trial</p>
+            <p class="reasons__advantages">World class customer service</p>
+            <p class="reasons__advantages">My Bose account management</p>
+            <p class="reasons__few-words">
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+            </p>
+          </section>
+        </section>
+      </article>
+
+      <article class="main__contacts contacts" id="contacts">
+        <p class="contacts__title">Contact us</p>
+
+        <article class="contacts__box">
+          <section
+            class="contacts__form"
+            data-aos="fade-up"
+            data-aos-duration="1200"
+          >
+            <form
```

Comment:
<img width="926" alt="Screenshot 2023-02-14 at 4 04 27 PM" src="https://user-images.githubusercontent.com/35467335/218761556-98f0c81d-ba07-4817-abb9-23dcf2742193.png">

You have wrong alignment for your contact us block, try to add only bottom margins to your paragraph elements.